eiffel-vocabulary/EiffelActivityFinishedEvent.md

```diff
@@ -0,0 +1,34 @@
+# EiffelActivityFinishedEvent
+The EiffelActivityFinishedEvent declares that a previously queued activity (declared by [EiffelActivityQueuedEvent](./EiffelActivityQueuedEvent.md)) has finished.
+
+## Data Members
+### data.outcome
+__Type:__ Object  
+__Required:__ Yes  
+__Description:__ The outcome of the activity.
+
+#### data.outcome.verdict
+__Type:__ String  
+__Required:__ Yes  
+__Legal values:__ SUCCESS, FAILURE, ERROR, ABORTED, TIMEOUT  
+__Description:__ A terse standardized verdict of the activity, designed to be machine readable.
+
+#### data.outcome.description
+__Type:__ String  
+__Required:__ No  
+__Description:__ A verbose description of the activity outcome, designed to provide human readers with further information.
+
+### data.persistentLogs
+__Type:__ Object[]  
+__Required:__ No  
```

Comment:
Intentional. The intent is: the object is optional, but if it's present, then the child may or may not be required. So it's a conditional Required if, you will.

I thought about it when typing it, realizing it might not be quite obvious :) Any suggestions?